platform/commonUI/browse/test/PaneControllerSpec.js

```diff
@@ -86,6 +97,26 @@ define(
                 // Tree should have collapsed
                 expect(controller.visible()).toBeFalsy();
             });
+
+            it("if url includes hideTree parameter then Tree is not visible", function () {
```

Comment:
`hideTree` and `hideInspector` are specific uses of hideParameter. You should be more specific: you are testing what hideParameter does.  e.g.

```javascript
describe("specifying hideParameter", function () {
    it("hides panel when parameter present in location", ...);
    it("does not hide panel when parameter not present in location", ...);
});
```